### PR TITLE
Release/1.22.0

### PR DESCRIPTION
**Pull request by rahulPalliyalil**
### What

This release pr contains the following:

- Switch to use es 7.10 client library
- Switch between transformers based on es version flag
- Unify aggregate clauses in both queries
- Add highlighting to transformer

### How to review

This is a release pr and while reviewing good to check with the contributors on each commit if this is intended. 

### Who can review

Anyone except me.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,11 +47,12 @@ environment variables, or with a link to a configuration guide.
...
 | AWS_REGION                   | eu-west-1                | The AWS region to use when signing requests with AWS SDK
 | AWS_SERVICE                  | "es"                     | The AWS service that the AWS SDK signing mechanism needs to sign a request
+| AWS_SIGNER                   | false                    | The AWS signer flag will determine if requests to Elasticsearch contain round tripper for signing requests
 | AWS_TLS_INSECURE_SKIP_VERIFY | false                    | This should never be set to true, as it disables SSL certificate verification. Used only for development
 | BIND_ADDR                    | :23900                   | The host and port to bind to
 | ELASTIC_SEARCH_URL	       | "http://localhost:9200"  | Http url of the ElasticSearch server. For Site Wide ElasticSearch this needs to be set to "http://localhost:11200".
 | GRACEFUL_SHUTDOWN_TIMEOUT    | 5s                       | The graceful shutdown timeout in seconds (`time.Duration` format)
-| SIGN_ELASTICSEARCH_REQUESTS  | false                    | Boolean flag to identify whether elasticsearch requests via elastic API need to be signed if elasticsearch cluster is running in aws
```
Comment (cookel2) [inline README.md:54]: If SIGN_ELASTICSEARCH_REQUESTS is no longer being used then it should also be removed from the config.go: https://github.com/ONSdigital/dp-search-api/blob/release/1.22.0/config/config.go#:~:text=SignElasticsearchRequests%20%20bool%20%20%20%20%20%20%20%20%20%20%60envconfig%3A%22SIGN_ELASTICSEARCH_REQUESTS%22%60

**Review by cookel2 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rahulPalliyalil) [reply README.md:54]: @eldeal : fixed and raised the pr: https://github.com/ONSdigital/dp-search-api/pull/119

Comment (rahulPalliyalil) [reply README.md:54]: @cookel2 This is now rebased to this release.

**Review by cookel2 — APPROVED** [review]
It works for me : )

**Pull request merged**